src/car/distance/DistanceCar.hpp

```diff
@@ -184,7 +184,7 @@ class DistanceCar : virtual public SimpleCar
      * car.overrideMotorSpeed(100, -100);
      * \endcode
      */
-    void overrideMotorSpeed(int firstMotorSpeed, int secondMotorSpeed) override;
+    virtual void overrideMotorSpeed(int firstMotorSpeed, int secondMotorSpeed) override;
```

Comment:
:warning: **modernize-use-override** :warning:
'virtual' is redundant since the function is already declared 'override'
```suggestion
    virtual void ```